.github/workflows/test-suite-nightly.yml

```diff
@@ -55,6 +55,9 @@ jobs:
       - name: 🌳 Display pod environment
         run: pod env
         working-directory: apps/bare-expo/ios
+      - name: 🥥 Update boost, RCT-Folly and SocketRocket pods in apps/bare-expo/ios
+        run: pod update boost RCT-Folly SocketRocket --no-repo-update
```

Comment:
sorry for late response. do you know why `npx pod-install` cannot deal with the updates?

Reply:
I'm not exactly sure why `npx pod-install` is unable to deal with these updates but it fails with the following error
>[!] CocoaPods could not find compatible versions for pod "RCT-Folly":
  In snapshot (Podfile.lock):
    RCT-Folly (from `../node_modules/react-native/third-party-podspecs/RCT-Folly.podspec`)
  In Podfile:
    RCT-Folly (from `../node_modules/react-native/third-party-podspecs/RCT-Folly.podspec`)
It seems like you've changed the version of the dependency `RCT-Folly` and it differs from the version stored in `Pods/Local Podspecs`.
You should run `pod update RCT-Folly --no-repo-update` to apply changes made locally.
Couldn't install: RCT-Folly. Updating the Pods project and trying again...
Command `pod install --repo-update` failed.
└─ Cause: This is often due to native package versions mismatching. Try deleting the 'ios/Pods' folder or the 'ios/Podfile.lock' file and running 'npx pod-install' to resolve.

Reply:
ok i figured out the problem. it's kinda problems from cocoapods and there's no better way in the meantime.
the flows of running pod-install are like:
1. pod-install runs `pod install`
2. cocoapods prompts `You should run 'pod update RCT-Folly --no-repo-update' to apply changes made locally.`
3. pod-install tries to run `pod update RCT-Folly --no-repo-update`
4. cocoapods prompts `You should run 'pod update boost --no-repo-update' to apply changes made locally.`
5. pod-install tries to run `pod update boost --no-repo-update`
6. cocoapods prompts `You should run 'pod update RCT-Folly --no-repo-update' to apply changes made locally.` again
7. pod-install tries to run `pod install --repo-update` as fallback
8. cocoapods still prompts `You should run 'pod update RCT-Folly --no-repo-update' to apply changes made locally.`
9. pod-install gives up for further attempts.

the case is that cocoapods does not well support multiple inconsistent pod versions. we have to run `pod update` a couple times until we know all of the outdated pods and run `pod update boost RCT-Folly SocketRocket --no-repo-update` at the same time. 🤦‍♂️